### PR TITLE
Clarifications to the push module

### PR DESCRIPTION
**Pull request by richvdh**
I think these are all just clarifications rather than changes.

Proabably easier to review the commits individually.

### CODE REVIEW

File: specification/modules/push.rst [hunk 1]
```diff
@@ -135,8 +135,9 @@ There are different "kinds" of push rules and each rule has an associated
 priority. Every push rule MUST have a ``kind`` and ``rule_id``. The ``rule_id``
 is a unique string within the kind of rule and its' scope: ``rule_ids`` do not
 need to be unique between rules of the same kind on different devices. Rules may
-have extra keys depending on the value of ``kind``. The different kinds of rule
-in descending order of priority are:
+have extra keys depending on the value of ``kind``.
+
+The different ``kind``\ s of rule in descending order of priority are:
```
Comment (ara4n) [inline specification/modules/push.rst:140]:
```suggestion
The different ``kind``\ s of rule, most important first, are:
```
I can never parse "ascending or descending order of priority", for whatever reason

**Comment by ara4n** [comment]
lgtm, although I wonder if we should incorporate more of my 'rationale block' from the summary I wrote the other day at https://docs.google.com/document/d/1Fnh8sT_8hmWAZ_Yh9NaB8HI5occNfR4KyQ_NVdnXJzo/edit.  It's definitely an improvement tho.

File: specification/modules/push.rst (hunk 1 above)
Comment (richvdh) [reply specification/modules/push.rst:140]:
```suggestion
The different ``kind``\ s of rule, in the order that they are checked, are:
```

**Comment by richvdh** [comment]
> lgtm, although I wonder if we should incorporate more of my 'rationale block' from the summary I wrote the other day at https://docs.google.com/document/d/1Fnh8sT_8hmWAZ_Yh9NaB8HI5occNfR4KyQ_NVdnXJzo/edit.

sorry, I'm not immediately finding anything matching the description of a "rationale block" there. 

I'm going to go ahead and land this. we can always iterate on it.

**Pull request merged**